lib/modules/dosomething/dosomething_northstar/dosomething_northstar.module

```diff
@@ -192,6 +212,10 @@ function dosomething_northstar_transform_user($user, $password = null) {
 
   // Set values in ns_user if they are set.
   foreach ($optional as $ns_key => $drupal_key) {
+    if (! isset($user->$drupal_key)) {
```

Comment:
actually @DFurnes no i cant, `$field = $user->$drupal_key;` was the code triggering the error. If i put this after that its pointless & the errors would show